figures/scripts/fig2_figS3-oncoprint-landscape.R

```diff
@@ -116,16 +125,49 @@ prep_histology_maf <- function(included_cancer_groups,
   # Fusions
   histology_fusion_df <- fusion_df %>%
     dplyr::filter(Tumor_Sample_Barcode %in% included_sample_ids)
-
-
-
+  
   # MAF object
   histology_maf_object <- prepare_maf_object(
     maf_df = histology_maf_df,
     cnv_df = histology_cnv_df,
     metadata = histologies_df_temp,
     fusion_df = histology_fusion_df
   )
+  
+  # Create a data frame that can be used to export this information
+  join_cols <- c("Hugo_Symbol", "Tumor_Sample_Barcode", "Variant_Classification")
+  
+  maf_export_df <- histology_maf_df %>%
+    dplyr::left_join(
+      histology_fusion_df, 
+      by = c(join_cols, "Variant_Type")) %>%
+    dplyr::left_join(histology_cnv_df,
+                     by = join_cols) %>%
```

Comment:
I guess we want full join here in case samples have no SNVs but only fusions or CNVs etc?